build.gradle

```diff
@@ -83,20 +89,15 @@ allprojects {
     }
 }
 
-task checkJavaVersion {
-    doLast {
-        if (JavaVersion.current().majorVersion.toInteger() < 11) {
-            throw new GradleException(
-                    "Your JDK (version ${JavaVersion.current()}) is too old to build Armeria. " +
-                    "Please check the build requirements at " +
-                    "https://armeria.dev/community/developer-guide/")
-        }
-    }
-}
-
 // Configure all Java projects
 configure(projectsWithFlags('java')) {
 
+    java {
+        toolchain {
+            languageVersion = JavaLanguageVersion.of(15)
```

Comment:
Shouldn't we specify `vendor` and `implementation` just in case it picks up a local JDK with a different vendor or implementation?

Reply:
Added. Thanks!